flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -171,7 +172,14 @@ public static void validateAwsConfiguration(Properties config) {
 		}
 
 		if (!config.containsKey(AWSConfigConstants.AWS_REGION)) {
```

Comment:
The new constructors make the easy path the right path.  We go through a lot of trouble at Amazon to make sure that the default constructors do the right thing with the minimal amount of effort.  Yet people still set things like region and auth manually when it is not only unnecessary but also a security, performance, and compliance risk.  Wherever we can we should try to follow the example of the AWS SDK and provide for using it correctly.  Overall, I would make the argument that using property files and statics isn't a best practice.  There really should be type safe POJOs and dependency injection in place for configuration of the consumer but that is a larger issue than I can take on right now.  The new constructors attempt to add some type safety while improving ease of use when operating in an Amazon environment.